PRIORITY_NOTES.md

```diff
@@ -0,0 +1,17 @@
+This file contains some notes on the Jenkins job priorities that are configured in this repository.
+
+The ROS buildfarm is currently setup with a Jenkins plugin that manages job priorities.  The priorities are inverse numerically based; that is, a lower number means a higher priority.  As of this writing, the buildfarm is configured to have a range of 1 to 109.  There are periodic maintenance jobs that use priorities 1 to 49, thus those priorities should never be used for distribution jobs.
+
+There are currently 6 classes of distribution jobs that the buildfarm runs.  Within each class, the ROS distributions should be in release order (newest ROS release has highest priority).  To make it so that we don't have to change the priorities on every release, we have defined some ranges for the classes, listed in the table below:
+
+| Job Type                     | Numeric Range |
+| ---------------------------- |:-------------:|
+| Pull Request                 | 50 - 59       |
+| Commit                       | 60 - 69       |
+| Source                       | 70 - 79       |
+| Basic Binary (amd64)         | 80 - 89       |
+| Documentation                | 90 - 99       |
+| Extended Architecture Binary | 100 - 109     |
```

Comment:
Since this only seems to need a range 10 numbers bigger I would suggest to shift all numbers by 10 down instead and modify the very few hard coded priorities of maintenance jobs to make room in the are from 40-49.

Reply:
So, I ended up moving back into the 1-99 range, and moved down some of the maintenance jobs in this PR: https://github.com/ros-infrastructure/ros_buildfarm/pull/500 .  At least we know we won't have any collisions for 5 years.